### PR TITLE
Fixes cats robusting larva

### PR DESCRIPTION
**Pull request by PhantomEpicness**
# About the pull request
Makes so huggers and larva can actually stand a chance against cats. 
Cats now do 25 damage and slugstun instead of instakill preys with clients

# Explain why it's good for the game
being instakilled isnt fun

# Testing Photographs and Procedure

<details>
<su

https://user-images.githubusercontent.com/42422230/221439636-81ac0dd6-1d13-40a7-9932-fc4aee42ce0a.mov

mmary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl:totalepicness
fix: Fixes cats now do 25 damage and a flicker stun to prey instead of instakilling them.
/:cl:

### CODE REVIEW

**Comment by carlarctg** [comment]
pusssy

**Comment by carlarctg** [comment]
carlarc approved

File: code/modules/mob/living/simple_animal/friendly/cat.dm [hunk 1]
```diff
@@ -43,6 +43,7 @@
 	black_market_value = 50
 	dead_black_market_value = 0
 	var/miaow_counter = 0
+	var/attack_damage = 15
```
Comment (Diegoflores31) [inline code/modules/mob/living/simple_animal/friendly/cat.dm:46]: you are not even using this var + it should be 25

File: code/modules/mob/living/simple_animal/friendly/cat.dm [hunk 2]
```diff
@@ -65,13 +66,27 @@
...
 					movement_target = null
 					stop_automated_movement = 0
+
+					animation_attack_on(livingprey)
+					flick_attack_overlay(livingprey, "slash")
+					if(!livingprey.client) // instakilling players bad
+						livingprey.splat(src)
+					else
+						livingprey.attack_animal(src)
+						livingprey.apply_damage(25,BRUTE)
```
Comment (Diegoflores31) [inline code/modules/mob/living/simple_animal/friendly/cat.dm:86]: use the var here i guess

**Review by harryob — CHANGES_REQUESTED** [review]
diego's right

**Review by harryob — APPROVED** [review]

**Pull request merged**